src/client/eppo-client.ts

```diff
@@ -211,7 +67,7 @@ export type FlagConfigurationRequestParameters = {
   skipInitialPoll?: boolean;
 };
 
-export default class EppoClient implements IEppoClient {
+export default class EppoClient {
```

Comment:
minor: Removing `IEppoClient` is an API change and requires a major version bump. Not sure if that consequence is intended.

Reply:
Yep, that's intended. It's also why this version is going to be `4.0.0`. I'll also mention this in the release notes.